src/conf_mode/protocols_igmp.py

```diff
@@ -79,6 +88,10 @@ def verify(igmp):
         return None
 
     if igmp['igmp_conf']:
+        # Check conflict with IGMP-Proxy
+        if igmp['igmp_proxy_conf']:
+            raise ConfigError(f"Does not possible to configure PIM/IGMP and IGMP-Proxy together!")
```

Comment:
This is ungrammatical. I would use "IGMP proxy and PIM cannot be both configured at the same time".